Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM golang:1.15
+FROM golang:1.16-alpine3.13
 
-RUN apt-get update && apt-get -y install zip && rm -rf /var/lib/apt/lists/*
+RUN apk add --no-cache --update git openssh-client make bash lftp zip
```

Comment:
@randomjunk this comes from https://github.com/Countingup/docker-node, I assume you figured out what Sempahore "needs" here?

Reply:
there's a very specific list:
https://docs.semaphoreci.com/ci-cd-environment/custom-ci-cd-environment-with-docker/#building-custom-docker-images

+ `make` which we require